resources/metadata/helmExecute.yaml

```diff
@@ -100,6 +110,9 @@ spec:
           - STEPS
         secret: true
         resourceRef:
+          - name: targetRepositoryCredentialsId
```

Comment:
if we expose this via a Jenkins credential, there should also be a proper Vault reference